src/lightning/pytorch/loops/evaluation_loop.py

```diff
@@ -45,6 +46,12 @@
 from lightning.pytorch.utilities.signature_utils import is_param_in_hook_signature
 
 
+@dataclass
+class RestartStage:
```

Comment:
lets maybe use `LightningEnum` for consistency?

Reply:
I'll make a small follow up PR with this change